### DISCUSSION TITLE
Compressed VisItDataCollection output --> "Draw" in VisIt Not Working ?

### DISCUSSION THREAD

**Discussion by j-signorelli** [mfem/mfem#3825] [Q&A]
Hi,

Whenever I try to "Draw" in VisIt using MFEM output from VisItDataCollection with `visit_dc->SetCompression(true)`. I get the following error in the terminal:

     MFEM abort: Unknown input mesh format: �
     terminate called after throwing an instance of 'mfem::ErrorException'
     what():  MFEM abort: Unknown input mesh format: �

When compression is set to `false`, I am able to load into VisIt perfectly fine. I was just wondering if anyone else has run into this issue / is this expected behavior? I am able to reload compressed VisIt output to restart simulations and would prefer to not need to use ASCII files.

Thank you for any input!

**Comment by tzanio** [comment]
@cyrush, do you know what could be causing this?
